lua/astrocommunity/markdown-and-latex/markdown-preview-nvim/init.lua

```diff
@@ -1,9 +1,29 @@
+local prefix = "<Leader>M"
+
+---@type LazySpec
 return {
   "iamcco/markdown-preview.nvim",
   build = function() vim.fn["mkdp#util#install"]() end,
   ft = { "markdown", "markdown.mdx" },
+  cmd = { "MarkdownPreviewToggle", "MarkdownPreview", "MarkdownPreviewStop" },
   init = function()
     local plugin = require("lazy.core.config").spec.plugins["markdown-preview.nvim"]
     vim.g.mkdp_filetypes = require("lazy.core.plugin").values(plugin, "ft", true)
   end,
+  dependencies = {
```

Comment:
The key mapping should be using this style
https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/code-runner/overseer-nvim/init.lua#L38-L53

opts will be a function and you can move the local prefix to the function